jsk_arc2017_baxter/euslisp/lib/arc-interface.l

```diff
@@ -446,28 +446,28 @@
     (send *ri* :wait-interpolation))
   (:pick-object-in-bin
     (arm bin &rest args)
-    (let (graspingp movable-region)
+    (let (pick-res movable-region)
       (setq movable-region (gethash bin bin-movable-regions-))
       (unless movable-region
         (ros::ros-error "[:pick-object-in-bin] No movable region for bin. Call :set-movable-region-for-bin first.")
         (return-from :pick-object-in-bin nil))
-      (setq graspingp (send* self :pick-object-with-movable-region arm movable-region args))
-      graspingp))
+      (setq pick-res (send* self :pick-object-with-movable-region arm movable-region args))
+      pick-res))
   (:pick-object-in-tote
     (arm &rest args)
-    (let (graspingp movable-region)
+    (let (pick-res movable-region)
```

Comment:
pick-res means pick-result.
This can be t, nil and :ik-fail
I wanted to distinguish picking no object and IK failue.